R/spaces_left_parentheses_linter.R

```diff
@@ -7,14 +7,15 @@ spaces_left_parentheses_linter <- function(source_file) {
 
       parsed <- source_file$parsed_content[id, ]
 
-      family_ids <- family(source_file$parsed_content, parsed$id)
-
-      types <- source_file$parsed_content[
-          source_file$parsed_content$id %in% family_ids,
-          "token"]
-
-      is_function <- length(family_ids) %!=% 0L &&
-        any(types %in% c("SYMBOL_FUNCTION_CALL", "FUNCTION"))
+      # content before parsed parentheses
+      before_parsed <- source_file$parsed_content[1:(id - 1), ]
```

Comment:
Could these lines be replaced by the `tokens_before` assignment in the closed_curly_linter?

https://github.com/jimhester/lintr/blob/56c35a3a387b136f4087b92c4992d70d39edb9e8/R/closed_curly_linter.R#L13-L15